endtoend/helpers/epochTimer.go

```diff
@@ -30,8 +30,8 @@ func (s *EpochTicker) Done() {
 	}()
 }
 
-// GetEpochTicker is the constructor for EpochTicker.
-func GetEpochTicker(genesisTime time.Time, secondsPerEpoch uint64) *EpochTicker {
+// New starts the EpochTicker.
+func New(genesisTime time.Time, secondsPerEpoch uint64) *EpochTicker {
```

Comment:
I disagree with this change - `helpers.New` does not convey information about what is being created. I would prefer `helpers.NewEpochTicker`.